### PR TITLE
Add new new raft symlink path to .gitignore

### PR DESCRIPTION
**Pull request by trxcllnt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,7 +76,7 @@ datasets/*
 cpp/doxygen/html
 
 # Raft symlink
-python/cugraph/raft
+python/cugraph/cugraph/raft
```
Comment (rlratzel) [inline .gitignore:79]:
```suggestion
python/cugraph/cugraph/raft
python/pylibcugraph/pylibcugraph/raft
```

**Review by rlratzel — CHANGES_REQUESTED** [review]
We have another python package now that uses raft the same way.

**Review by rlratzel — APPROVED** [review]

**Comment by rlratzel** [comment]
Thanks for this update!

**Review by BradReesWork — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**